Source/Core/Core/FifoPlayer/FifoAnalyzer.cpp

```diff
@@ -4,10 +4,10 @@
 
 #include "Core/FifoPlayer/FifoAnalyzer.h"
 
-#include <algorithm>
```

Comment:
Why did you remove this `#include`? `AnalyzeCommand` still uses `std::accumulate` which comes from here.

Reply:
`std::accumulate` is located within `<numeric>`

Reply:
Got carried away by the [cppreference.com URL](https://en.cppreference.com/w/cpp/algorithm/accumulate), which has `algorithm` in it (for the algorithm library, not the `algorithm` header, as it turns out...)